src/libcollections/vec_deque.rs

```diff
@@ -1232,9 +1232,8 @@ impl<T> VecDeque<T> {
         self.pop_front()
     }
 
-    /// Inserts an element at `index` within the `VecDeque`. Whichever
-    /// end is closer to the insertion point will be moved to make room,
-    /// and all the affected elements will be moved to new positions.
+    /// Inserts an element within the `VecDeque` at index `index`. All elements previously with
+    /// indices greater than or equal to `index` will have their indices incremented by one.
```

Comment:
My hesitation with that comment is that (at least behind the scenes in memory), `VecDeque::insert` doesn't necessarily shift all the following items to the back like it does with `Vec::insert`. Though, if we're not talking about what happens in-memory it is true as far as indices/positions go, if that makes any sense. What do you think?